inc/task.class.php

```diff
@@ -1690,7 +1705,7 @@ function csvExport($params = []) {
       ];
       $params = array_merge($default_params, $params);
 
-      $includeoldjobs    = $_SESSION['glpi_plugin_fusioninventory']['includeoldjobs'];
+      $includeoldjobs = $_SESSION['glpi_plugin_fusioninventory']['includeoldjobs'];
       $agent_state_types = ['prepared', 'cancelled', 'running', 'success', 'error' ];
```

Comment:
In line 1040 we have another variable called $agent_state_types with other states... for example 'agent_prepared' and here 'prepared'
Could'nt we just normalize that and use only one constant value ?